src/err/mod.rs

```diff
@@ -476,6 +476,15 @@ impl PyErr {
         unsafe { ffi::PyErr_Restore(ptype, pvalue, ptraceback) }
     }
 
+    /// Reports the error as unraisable.
+    ///
+    /// This calls `sys.unraisablehook()` using the current exception and obj argument.
```

Comment:
I'd like this documentation to be fleshed out a bit more. e.g.: What does an unraisable error do? when you you use it? when shouldn't you use it? An example would also be appreciated :)